lang/cs/Org.Apache.REEF.Driver/DriverConfigGenerator.cs

```diff
@@ -41,6 +41,11 @@ public static class DriverConfigGenerator
         public const string NameServerConfigFile = "nameServer.config";
         public const string UserSuppliedGlobalLibraries = "userSuppliedGlobalLibraries.txt";
 
+        /// <summary>
+        /// The bridge JAR name.
+        /// </summary>
+        public const string JavaBridgeJarFileName = "reef-bridge-java-0.14.0-SNAPSHOT-shaded.jar";
```

Comment:
I think we don't want to have file name constants in Constants.cs, so some of them went to REEFFilenames, but this one seems to make more sense here, closer to where it is used (which is just a single place).